numpy/core/_dtype_ctypes.py

```diff
@@ -22,8 +22,6 @@ class DummyStruct(ctypes.Structure):
 * PEP3118 cannot represent unions, but both numpy and ctypes can
 * ctypes cannot handle big-endian structs with PEP3118 (bpo-32780)
 """
-import _ctypes
-import ctypes
```

Comment:
You should add a comment here noting that ctypes is deliberately not imported at the top of the file.